scripts/dataset_processing/spoken_wikipedia/run.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+## Download the Spoken Wikipedia corpus for English
+## Note, that there are some other other languages available
+wget https://corpora.uni-hamburg.de/hzsk/de/islandora/object/file:swc-2.0_en-with-audio/datastream/TAR/en-with-audio.tar .
+tar -xvf en-with-audio.tar
+
+##  We get a folder English with 1339 subfolders, each subfolder corresponds to a Wikipedia article. Example:
+##  ├── Universal_suffrage
+##  │   ├── aligned.swc
+##  │   ├── audiometa.txt
+##  │   ├── audio.ogg
+##  │   ├── info.json
+##  │   ├── wiki.html
+##  │   ├── wiki.txt
+##  │   └── wiki.xml
+
+##  We will use two files: audio.ogg and wiki.txt
+
+##  Some folders have multiple .ogg files, this will be handled during preprocess.py. Example:
+##  |── Universe
+##  │   ├── aligned.swc
+##  │   ├── audio1.ogg
+##  │   ├── audio2.ogg
+##  │   ├── audio3.ogg
+##  │   ├── audio4.ogg
+##  │   ├── audiometa.txt
+##  │   ├── info.json
+##  │   ├── wiki.html
+##  │   ├── wiki.txt
+##  │   └── wiki.xml
+
+##  Some rare folders are incomplete, these will be skipped during preprocessing.
+
+## path to NeMo repository, e.g. /home/user/nemo
+NEMO_PATH=
+
+INPUT_DIR="english"
+OUTPUT_DIR=${INPUT_DIR}_result
+
+rm -rf $OUTPUT_DIR
+rm -rf ${INPUT_DIR}_prepared
+mkdir ${INPUT_DIR}_prepared
+mkdir ${INPUT_DIR}_prepared/audio
+mkdir ${INPUT_DIR}_prepared/text
+python ${NEMO_PATH}/scripts/dataset_processing/spoken_wikipedia/preprocess.py --input_folder ${INPUT_DIR} --destination_folder ${INPUT_DIR}_prepared
+
+
+MODEL_FOR_SEGMENTATION="QuartzNet15x5Base-En" 
```

Comment:
This is a duplication of https://github.com/NVIDIA/NeMo/blob/main/tools/ctc_segmentation/run_segmentation.sh and https://github.com/NVIDIA/NeMo/blob/main/tools/ctc_segmentation/run_filter.sh. How about calling these bash scripts with appropriate args from yours?

Reply:
done